src/internal/kopia/backup_bases_test.go

```diff
@@ -151,12 +155,97 @@ func (suite *BackupBasesUnitSuite) TestMinBackupVersion() {
 					},
 				},
 			},
-			expectedVersion: version.NoBackup,
+			expectedBackupVersion: version.NoBackup,
+			expectedAssistVersion: 0,
+		},
+		{
+			name: "Assist and Merge Bases, min merge",
+			bb: &backupBases{
+				mergeBases: []BackupBase{
+					{
+						Backup: &backup.Backup{
+							Version: 4,
```

Comment:
nit: for the set of newly added tests could you please make sure all backup versions are disjoint between assist and merge bases? This will make sure there's no accidental overlap in how the implementation handles things